os/board/Kconfig

```diff
@@ -191,6 +192,11 @@ config ARCH_IRQBUTTONS
 	---help---
 		"Support interrupts on button presses and releases."
 
+config ARCH_HAVE_RAM_KERNEL_TEXT
+	default n
```

Comment:
Maybe this causes menuconfig error or warning.
```
config ARCH_HAVE_..
     bool
     default n
     ---help---
           To support ...
````
You should do like above.

Reply:
done